processing/src/main/java/io/druid/segment/incremental/IncrementalIndex.java

```diff
@@ -240,6 +241,7 @@ public ColumnCapabilities getColumnCapabilities(String columnName)
   private final List<DimensionDesc> dimensionDescsList;
   private final Map<String, ColumnCapabilitiesImpl> columnCapabilities;
   private final AtomicInteger numEntries = new AtomicInteger();
+  private final AtomicLong sizeInBytes = new AtomicLong();
```

Comment:
Consider calling this `bytesInMemory` for consistency with the config name. So it's clear that they're related.

Reply:
done